floorp.ftl

```diff
@@ -571,29 +571,29 @@ workspace-icon-chill =
  .label = Private
 
 # CSK
-floorp-CSK-title = Custom Shortcut Keys
-floorp-CSK-description = Customize the shortcut keys of { -brand-short-name }. Floorp provides more 80 shortcut keys. Reboot { -brand-short-name } to apply these settings. Duplicates do not work.
+floorp-CSK-title = Custom keyboard shortcuts
+floorp-CSK-description = Customize { -brand-short-name }'s keyboard shortcuts. Floorp provides more than 80 customizable keyboard shortcuts - restart { -brand-short-name } to apply these settings. Duplicate keyboard shortcuts will not work.
 disable-fx-actions =
- .label = Disable Firefox's default shortcut keys
+ .label = Disable built-in keyboard shortcuts
```

Comment:
In fact, This feature does not disable, Workspaces shortcut key.

Is this correct?

Reply:
Ah, in that case...

Reply:
I think both are good.

I'm not native. You can choice.

Reply:
Nah, built-in implies Floorp's shortcuts too so people would assume such and consider it a bug if I keep it that way.